### ISSUE TITLE
Unable to install dcgm, dcgm-exporter in kubevirt vm-passthrough mode

### ISSUE THREAD

**Issue by rokkiter** [NVIDIA/gpu-operator#711]
_The template below is mostly useful for bug reports and support questions. Feel free to remove anything which doesn't apply to you and add more information where it makes sense._

_**Important Note:  NVIDIA AI Enterprise customers can get support from NVIDIA Enterprise support. Please open a case [here](https://enterprise-support.nvidia.com/s/create-case)**._

### 1. Quick Debug Information
* OS/Version(e.g. RHEL8.6, Ubuntu22.04): Ubuntu22.04
* Kernel Version:
* Container Runtime Type/Version(e.g. Containerd, CRI-O, Docker): Containerd
* K8s Flavor/Version(e.g. K8s, OCP, Rancher, GKE, EKS): v1.25.6
* GPU Operator Version: latest

### 2. Issue or feature description
I want to monitor GPUs in kubevirt passthrough mode, but nodes set to vm-passthrough don't have dcgm, dcgm-export installed. is there any way to implement monitoring GPUs in kubevirt passthrough mode?

### 3. Steps to reproduce the issue
Refer to this document to build the kubevirt vm-passthrough environment.
https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/latest/gpu-operator-kubevirt.html

### 4. Information to [attach](https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/) (optional if deemed irrelevant)

 - [ ] kubernetes pods status: `kubectl get pods -n OPERATOR_NAMESPACE`
 - [ ] kubernetes daemonset status: `kubectl get ds -n OPERATOR_NAMESPACE`
 - [ ] If a pod/ds is in an error state or pending state `kubectl describe pod -n OPERATOR_NAMESPACE POD_NAME`
 - [ ] If a pod/ds is in an error state or pending state `kubectl logs -n OPERATOR_NAMESPACE POD_NAME --all-containers`
 - [ ] Output from running `nvidia-smi` from the driver container: `kubectl exec DRIVER_POD_NAME -n OPERATOR_NAMESPACE -c nvidia-driver-ctr -- nvidia-smi`
 - [ ] containerd logs `journalctl -u containerd > containerd.log`

Collecting full debug bundle (optional):

```
curl -o must-gather.sh -L https://raw.githubusercontent.com/NVIDIA/gpu-operator/master/hack/must-gather.sh 
chmod +x must-gather.sh
./must-gather.sh
```
**NOTE**: please refer to the [must-gather](https://raw.githubusercontent.com/NVIDIA/gpu-operator/master/hack/must-gather.sh) script for debug data collected.

This bundle can be submitted to us via email: **operator_feedback@nvidia.com**

**Comment by cdesiniotis** [comment]
@rokkiter we do not have a solution available today for gathering GPU metrics on nodes configured for GPU passthrough.

**Comment by raghavendra-rafay** [comment]
@cdesiniotis, Is there a plan to add this support anytime soon?

**Comment by cdesiniotis** [comment]
@raghavendra-rafay no, we don't have plans in the near term. Since there are no NVIDIA drivers running on the host when GPUs are prepared for passthrough, we cannot run dcgm on the worker node. It has to run alongside the workload (VM) -- and how to orchestrate this and propagate metrics back to the host is not straightforward.

**Comment by shallyg05** [comment]
Closing this RFE as it is not planned. GPU Operator does not perform any operation in the Kubevirt VM itself. We recommend to use appropriate installation software for this purpose.

**Comment by francisguillier** [comment]
We have added the ask into our backlog.
However, we currently have no plans to implement these features, as they don't align with the primary use cases we're seeing from the majority of our users.
